scanners/zap-advanced/.helmignore

```diff
@@ -5,8 +5,13 @@
 .DS_Store
 
 parser/
-scanner/
-examples/
```

Comment:
Examples should be excluded...

Reply:
Ah, forgot a new line, the `examples/` exclude was accidentally put into the same line as the `.idea/` making both worthless. 👍